test/functionalTest/cases/0000_content_related_headers/accept_fail.test

```diff
@@ -0,0 +1,305 @@
+# Copyright 2013 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Not really a new file, but a wrongly detected modification + mv.

NTC (just informative)